housekeeper/archive/core.py

```diff
@@ -35,5 +38,6 @@ def get_archiveassets(run_id):
     for archive_type in ("data", "result"):
         category = "archive-{}".format(archive_type)
         query = api.assets(run_id=run_id, category=category)
-        asset = query.first()
-        yield archive_type, asset
+        assets = query.all()
+        for asset in assets:
+            yield archive_type, asset
```

Comment:
I'm not sure changing this from returning one to everything is a good idea tho. I need it for now, as to return both compilation and compilation key. But what when we have sent a compilation multiple times to PDC? What do you think?